circuits/cpp/src/aztec3/circuits/rollup/root/.test.cpp

```diff
@@ -212,6 +212,10 @@ TEST_F(root_rollup_tests, native_root_missing_nullifier_logic)
     MemoryTree historic_data_tree = MemoryTree(PRIVATE_DATA_TREE_ROOTS_TREE_HEIGHT);
     MemoryTree historic_contract_tree = MemoryTree(CONTRACT_TREE_ROOTS_TREE_HEIGHT);
 
+    // Historic trees are initialised with an empty root at position 0.
+    historic_data_tree.update_element(0, data_tree.root());
+    historic_contract_tree.update_element(0, contract_tree.root());
```

Comment:
`historic_l1_to_l2_msgs_tree` is initialised in both other instances where these trees are initialised, why not here?